src/splitInteger.test.js

```diff
@@ -8,12 +8,32 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(typeof splitInteger()).toBe('object');
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
-
+    expect(splitInteger(8, 1)).toStrictEqual([8]);
   });
-  
+
   // write more tests here
+
+  it(`the parts of the 'num' should be as close as possible`, () => {
+    expect(splitInteger(17, 4)).toStrictEqual([4, 4, 4, 5]);
+  });
+
+  it(`the integers in the result array should be ordered ASC`, () => {
+    expect(splitInteger(32, 6)).toStrictEqual([5, 5, 5, 5, 6, 6]);
+  });
+
+  it(`should return an empty array if num = 3, parts = 'string'`, () => {
+    expect(splitInteger(3, 'string')).toStrictEqual([]);
+  });
+
+  it(`should return an empty array if num = 'string', parts = 5`, () => {
+    expect(splitInteger('string', 5)).toStrictEqual([]);
+  });
+
+  it(`should return an empty array if num = 'string', parts = 'string'`, () => {
+    expect(splitInteger('string', 'string')).toStrictEqual([]);
+  });
```

Comment:
The description of the function says:
`the input to your function will always be valid`

`'string'` is an invalid input, so these tests are not needed.